app/models/katello/content_view.rb

```diff
@@ -316,6 +317,14 @@ def duplicate_puppet_modules
       counts.select { |_k, v| v > 1 }.keys
     end
 
+    def check_non_composite_components
+      unless composite?
+        unless components.blank?
```

Comment:
Bah, stupid unlesses confusing me, could we combine those into a single statement?